libraries/SITL/SITL.cpp

```diff
@@ -105,6 +105,24 @@ const AP_Param::GroupInfo SIM::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("CAN_SRV_MSK",   29, SIM,  can_servo_mask, 0),
 
+#if HAL_NUM_CAN_IFACES > 0
+    // @Param: CAN_TYPE0
+    // @DisplayName: transport type for first interface
+    // @Description: transport type for first interface
+    // @Values: 0:MulticastUDP,1:SocketCAN
+    // @User: Advanced
+    AP_GROUPINFO("CAN_TYPE0", 30, SIM,  can_transport[0], uint8_t(CANTransport::MulticastUDP)),
```

Comment:
Is the related to P1 and P2.. if yes perhaps we should change the param to be CAN_TYPE1?

Reply:
done